lib/bb/server/process.py

```diff
@@ -392,10 +392,10 @@ def __init__(self, connection, recv):
 
     def runCommand(self, command):
         self.connection.send(command)
-        if not self.recv.poll(30):
-            logger.info("No reply from server in 30s")
-            if not self.recv.poll(30):
-                raise ProcessTimeout("Timeout while waiting for a reply from the bitbake server (60s)")
+        if not self.recv.poll(300):
+            logger.info("No reply from server in 300s")
+            if not self.recv.poll(300):
+                raise ProcessTimeout("Timeout while waiting for a reply from the bitbake server (300s)")
```

Comment:
Why not just use the `self.maxuiwait` variable here, instead of magic-numbering `30`/`300`?

Also, it seems like this timeout should be a bitbake variable. Would it take a lot of effort to variablize it? I think upstream would accept if we did that.

Reply:
I'm not sure if poll timeout is same as maxuiwait although they could have same values. I can look into this and work on making this a bitbake variable. I have no idea how long this would take (hopefully not more than a day). Can we merge the current changes in the meantime so builds don't keep failing?

Reply:
>Can we merge the current changes in the meantime so builds don't keep failing?

Yeah; that's fine. Just put a revert-commit in the variablization patchset.